### DISCUSSION TITLE
Might be a silly question, but how is work on Fish Shell funded?

### DISCUSSION THREAD

**Discussion by mrchrisadams** [fish-shell/fish-shell#10877] [General]
Hi there,

I recently switched to the fish shell again after trying it around ten years ago, and I forgot how nice the experience of using it was. 

It's now my default shell everywhere I can use it - thank you for making such a nice piece of software!

I know lots of people like me now rely on Fish, so I should probably ask - how is the project funded? Is anyone employed to work on it, or is there any one actively looking for grants to fund it? Is there any formally recommended way to support it financially?

I ask as it feels like it might be a good candidate for funding with something like the [German Sovereign Tech Fund's fellowship programme](https://www.sovereign.tech/programs/fellowship), as it's widely used, and for many people, it counts as a critical open source component in their tech stack. It's also reliant on a fairly small team, where the source of funding was not immediately obvious.

Previous funding has gone to projects like Wireguard, Curl, and a [fair few others recognisable OSS projects](https://www.sovereign.tech/tech)

**Comment by faho** [comment]
> Is anyone employed to work on it, or is there any one actively looking for grants to fund it? Is there any formally recommended way to support it financially?

That's a "no", a "no" and another "no".

We do this as a hobby and are not looking for funding.

**Comment by krobelus** [comment]
thanks for the pointer, I'm definitely interested
